CHANGELOG.md

```diff
@@ -10,6 +10,17 @@ command line for details.
 
 ## 0.8
 
+### [0.8.1] - 2019-02-27
+
+- Provide better error messages
+- Allow auth scope to be array or strings
+- GitHubOAuthenticator: More efficient org_whitelist check
```

Comment:
Thanks @amit1rrr. 

I would recommend putting keywords in backticks. For example,

```
- `GitHubOauthenticator`: More efficient `org_whitelist` check
```

would render as:

`GitHubOauthenticator`: More efficient `org_whitelist` check

The user will find this makes it easier to scan the changelog.

Reply:
Good idea. Done.